src/dynarec/la64/dynarec_la64_f0.c

```diff
@@ -62,11 +62,35 @@ uintptr_t dynarec64_F0(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int ni
                 emit_add32(dyn, ninst, rex, ed, gd, x3, x4, x5);
             } else {
                 addr = geted(dyn, addr, ninst, nextop, &wback, x2, x1, &fixedaddress, rex, LOCK_LOCK, 0, 0);
-                MARKLOCK;
-                LLxw(x1, wback, 0);
-                ADDxw(x4, x1, gd);
-                SCxw(x4, wback, 0);
-                BEQZ_MARKLOCK(x4);
+                if (rex.w) {
+                    if (!ALIGNED_ATOMICxw) {
```

Comment:
We don't support this in rv64 and la64 for now.

Reply:
But we can. It's just an optimisation that doesn't generate unaligned code (and save some space)

Reply:
all opcodes ... at once? Tha seems like a lot of work? why not doing it gradualy?

Reply:
I think it's fine to have partial implementation of stuffs that are optimisation, but I leave that to you @ksco . If you preer all-once instead of gradual, no pb.

Reply:
Sure, I don’t have strong opinions here.